.gitignore

```diff
@@ -2,3 +2,5 @@
 node_modules
 .idea
 .vscode
+
+*storybook.log
```

Comment:
Please add EOF line break (and [you should probably setup your IDE](https://weare.frontify.com/document/42#/developer-setup/ide:12299))

Reply:
added and ide config changed